packages/gestalt/src/ComboBox.js

```diff
@@ -151,6 +151,12 @@ type Props = {
    * An object representing the zIndex value of the ComboBox list box. Learn more about [zIndex classes](https://gestalt.pinterest.systems/web/zindex_classes)
    */
   zIndex?: Indexable,
+  /**
+   * *EXPERIMENTAL:* Whether to hide ComboBox when reference element gets out of viewport.
+   */
+  hideWhenReferenceHidden?: boolean,
+  // Whether to trap focus inside ComboBox when opened.
+  __disableFocusTrap?: boolean,
```

Comment:
nit: this only needs a single underscore prefix to be excluded from the published props table